app/_includes/plugins/example.md

```diff
@@ -12,6 +12,16 @@
 
 {% endif %}
 
+{% if page.example.variables %}
```

Comment:
This if statement doesn't seem to work. No matter if the example has env variables or not, the header still shows up. For example, here's a plugin without prereqs or env variables set: https://deploy-preview-545--kongdeveloper.netlify.app/plugins/acl/examples/allow-consumer-groups/

Reply:
🤦 liquid is tricky when it comes to arrays and emptiness, I've fixed it now.